src/Batteries.purs

```diff
@@ -1775,6 +1782,74 @@ import Math
   , tan
   , (%)
   )
+import Optic.Core
+  ( (..)
+  )
```

Comment:
The core module actually exports a lot more than this, but I think it makes more sense to export from the other modules (like getter).